src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[]
+};
+
+export const TabsPages: FC<Props> = ({ tabs }) => {
+  const { tabId = 0 } = useParams();
```

Comment:
The default value of `tabId` should be a string instead of a number since useParams returns an object with string values